docs/helpers/deploy-instance-demo.sh.template

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Copyright 2023 Red Hat Inc.
```

Comment:
not sure what is right for the date here... if the entire file is copy/pasted from elsewhere then leave it as 2023? otherwise lets update?